network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html

```diff
@@ -6,13 +6,15 @@
     {% if uses_menu %}
         {% block minisite_title %}
         {% endblock %}
+        {% if mini_site_title %}
```

Comment:
This results in an empty div otherwise.

<img width="1146" alt="Screenshot 2024-02-13 at 13 00 27" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/35934884/a06fb45a-5ad5-47b6-8146-509e8d15ac0e">

Reply:
Thanks for the note!